operator/pkg/reconciliation/construct_service.go

```diff
@@ -144,7 +157,23 @@ func newEndpointsForAdditionalSeeds(dc *api.CassandraDatacenter) *corev1.Endpoin
 
 	utils.AddHashAnnotation(&endpoints)
 
-	return &endpoints
+	return &endpoints, nil
+}
+
+func resolveAddress(hostname string) ([]string, error) {
+	ips, err := net.LookupIP(hostname)
+	if err != nil {
+		return []string{}, err
+	}
+	ipStrings := make([]string, 0, len(ips))
+	for _, ip := range ips {
+		// Exclude IPv6 addresses
+		if ip.To4() != nil {
+			ipStrings = append(ipStrings, ip.String())
+		}
+	}
+
+	return ipStrings, nil
```

Comment:
am I wrong in saying it should return only one IP seeing how you do it ?

Reply:
Yes, since net.LookupIP could resolve to multiple addresses (according to the doc).